.gitignore

```diff
@@ -24,3 +24,5 @@ openqa.log
 t/data/openqa/factory/hdd/00099963-hdd_image2.qcow2
 t/data/openqa/factory/hdd/hdd_image.qcow2
 t/diagram-v*.png
+assets/cache/
+assets/assetpack.db
```

Comment:
I discourage adding cache and assetpack.db to gitignore. It will impossible to know if AssetPack for a coworker/another environment will have the same result if you don't have the same cache files to work with. Also, assetpack.db contains crucial information wich is required when processing the assets.